codespell_lib/data/dictionary.txt

```diff
@@ -24289,8 +24289,16 @@ pytnon->python
 pytohn->python
 pyton->python
 pytyon->python
+qest->quest
+qests->quests
+qeuest->quest
```

Comment:
please add `quests` as another answer

Reply:
I'd better fix the request one too!

Reply:
Actually sorry why @luzpaz , I don't really see why this suggests quests rather than the singular. I've got the plural versions of those typos in already.

Reply:
You're right. My mistake.